code/game/gamemodes/meteor/meteors.dm

```diff
@@ -468,7 +468,9 @@ var/list/blob_candidates = list()
 
 	playsound(user, 'sound/weapons/rocket.ogg', 100)
 
-	new projectile_type(get_turf(src), get_turf(A))
+	var/obj/item/projectile/meteor/in_chamber = new projectile_type(get_turf(src), get_turf(A))
+
+	log_attack("[user.name] ([user.ckey]) fired \the [src] (proj:[in_chamber.name]) at [A] [ismob(A) ? "([A:ckey])" : ""] ([A.x],[A.y],[A.z])" )
```

Comment:
And I'd rather the meteor gun was actually a gun, but it isn't, and add_logs is useful for doing everything needed for proper logging, including LAssailant, which you don't do here.

So, add_logs.